### ISSUE TITLE
Leapp does not detect qemu64 CPU incompatibility with CentOS-9

### ISSUE THREAD

**Issue by MigNov** [oamg/leapp-repository#1105]
**Actual behavior**
When you run leapp on CentOS-7/CentOS-8 upgrade to CentOS-9 it goes on but CentOS-9 kernel does not boot.
The CentOS-8 warns about CPU fmaily 15, model 107, stepping 1 being without upstream testing.
The CentOS-9 does not even boot on that CPU.

The CPU is qemu64.

**To Reproduce**

Run Leapp normally and it's working fine however Leapp does not report any error (so nothing to report) but after upgrade to CentOS-9 the system crashes on kernel panic.

**Expected behavior**

Leapp prevents the upgrade on unsupported CPU - i.e. device_driver_deprecation_data.json has information about CPU device_id "x86_64:intel:15:107" (in this case) being unsupported.

**System information**

- original OS: CentOS-7
```
Linux centos7 3.10.0-1127.el7.x86_64 #1 SMP Tue Mar 31 23:36:51 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="https://www.centos.org/'
```

Thanks,
Michal

**Comment by pirat89** [comment]
@MigNov  Hi! Thank you for the report. I expect the problem has been caused by an old microarchitecture of the cpu. Can you share with us `lscpu` output of the affected system? If I am right, the problem is fixed separately from the data file already in upstream.

We have introduced the check for this by the following PR: https://github.com/oamg/leapp-repository/pull/1059 - available in current upstream devel builds. However our versions of packages do not allow upgrades of CentOS systems yet, so it would require some manual changes in the code to make it working.

**Comment by MigNov** [comment]
Hi @pirat89 ,
sure, the `lscpu` output is as follows:

```
[root@centos7 ~]# lscpu
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                2
On-line CPU(s) list:   0,1
Thread(s) per core:    1
Core(s) per socket:    1
Socket(s):             2
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            15
Model:                 107
Model name:            QEMU Virtual CPU version 2.5+
Stepping:              1
CPU MHz:               2400.000
BogoMIPS:              4800.00
Hypervisor vendor:     KVM
Virtualization type:   full
L1d cache:             32K
L1i cache:             32K
L2 cache:              4096K
L3 cache:              16384K
NUMA node0 CPU(s):     0,1
Flags:                 fpu de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 syscall nx lm constant_tsc nopl xtopology eagerfpu pni cx16 x2apic hypervisor lahf_lm
```

Thanks,
Michal

**Comment by pirat89** [comment]
@MigNov  Thanks. I can confirm it's due to old microarchitecture. Following flags are missing:
```
['popcnt', 'sse4_1', 'sse4_2', 'ssse3']
```

So the issue is fixed in the current upstream code by #1059. The fix will be part of the next version.

**Comment by MigNov** [comment]
OK, thanks for the information.

Michal
